RecoTracker/LSTCore/interface/alpaka/Constants.h

```diff
@@ -83,23 +83,26 @@ namespace SDL {
   ALPAKA_STATIC_ACC_MEM_GLOBAL constexpr float widthPS = 0.01;
   ALPAKA_STATIC_ACC_MEM_GLOBAL constexpr float pt_betaMax = 7.0;
```

Comment:
Should we switch to the recommended naming starting with 'k' (e.g. `kWidthPS`) and avoid snake case since they might bring it up during review?

Reply:
I merged before resolving this.
The renaming of a few constants should be relatively easy to add in another PR to `batch4`

Reply:
I will start working on this, to be added together with the removal of vector copies.